pystiche_papers/johnson_alahi_li_2016/_modules.py

```diff
@@ -202,6 +208,19 @@ def encoder(instance_norm: bool = True,) -> pystiche.SequentialModule:
 def decoder(
     impl_params: bool = True, instance_norm: bool = True,
 ) -> pystiche.SequentialModule:
+    r"""Decoder part of the Transformer from :cite:`JAL2016` .
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper.
+
+    If ``impl_params`` is ``True``, a tanh with a constant factor of 150 is used instead
```

Comment:
It is not clear what constant factor means.

1. 150 * tanh(x)
2. tanh(150 * x)

Also

```suggestion
    If ``impl_params is True``, a :func:`~torch.tanh` with a constant factor of 150 is used instead
```